### DISCUSSION TITLE
new status icons

### DISCUSSION THREAD

**Discussion by Anthonyg5005** [vrcx-team/VRCX#821] [General]
after upgrading to the latest stable version, I noticed new status icons. They obviously are inspired by discord but I feel like they should be different. just because someone is on ask me it shouldn't look like they're afk or something, also just because someone is on quest doesn't mean it should look like they're on phones, maybe a meta logo like icon would fit better. I feel like the idle icon should maybe be on active instead of ask me or something. at least an option changing the looks of it could be added to the appearance settings.

**Comment by Myrkie** [comment]
the icons were not entirely inspired by discord, they were changed to be more visually distinct for people with color blindness, thus why a lot of them have chunks missing out of them to be a more distinct shape especially while in VR with low resolution HMDs.

**Reply by Anthonyg5005** [reply]
That makes more sense. I still feel like maybe there should be some sort of setting to change it back or something, it's more visually confusing to me. It's not much too of a problem though, which is why I chose discussion instead of opening an issue

**Reply by Myrkie** [reply]
https://github.com/vrcx-team/VRCX/issues/822
